pkg/storaged/btrfs/device.jsx

```diff
@@ -41,6 +41,7 @@ export function make_btrfs_device_card(next, backing_block, content_block, block
     const btrfs_card = new_card({
         title: _("btrfs device"),
         location: label || uuid,
+        location_goto: ["btrfs-volume", uuid],
```

Comment:
I'm not sure `location_goto` is super descriptive, what is it supposed to do?

Reply:
I agree.  This is the value that should be passed to `cockpit.location.go` to navigate to the thing named by the `location` field above.

The `location` field shows up in the "Location" column in the tables. For a filesystem its the mount point of that filesystem, for a LVM2 physical volume it is the name of the volume group that it belongs to, etc.

Thus, the "Location" column header itself is not ideal. Maybe "Used for" is better? I am not sure about the history of how we decided this. If you have any ideas, tat would be great.

So, technically, we should use `location_location` instead of `location_goto`. But `location_location` is also aweful, no? Or `used_for_location` if we rename the `location` field to `used_for`.

Reply:
The best would be to consolidate it into `location` prop itself but I'm unsure how much work that would be something like this (though I would likely split this out to be an interface for less human error)
```
location: {
  label: label || uuid,
  // OR
  name: label || uuid,
  to: ["btrfs-volume", uuid]
}
```

In routing props et. al. it is common to see a `to` prop. So I would be fine with `location_to` if that is necessary. WDYT?

Reply:
> The best would be to consolidate it into `location` prop itself

That's a good idea!

It's totally doable, I'll give it a shot. It will probably also mean cleaning up the "page" data structure. This would be really nice with TypeScript...

Reply:
Done!